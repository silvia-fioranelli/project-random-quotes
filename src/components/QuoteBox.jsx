function QuoteBox() {
  return (
    <div class="card text-center" style={{ width: '30rem' }}>
      <div class="card-body">
        <blockquote class="blockquote">
          <p class="mb-0">
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer
            posuere erat a ante.
          </p>
          <br/>
          <footer class="blockquote-footer">
            Author
          </footer>
        </blockquote>
        <hr />
        <button className="btn btn-light">
          <i className="fa fa-refresh"></i>
        </button>
      </div>
    </div>
  );
}

export default QuoteBox;
