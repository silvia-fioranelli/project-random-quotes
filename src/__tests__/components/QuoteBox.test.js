import { shallow } from 'enzyme';
import QuoteBox from '../../components/QuoteBox';

describe('QuoteBox', () => {
  describe('rendering', () => {
    test('component QuoteBox should render', () => {
      const wrapper = shallow(<QuoteBox />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
