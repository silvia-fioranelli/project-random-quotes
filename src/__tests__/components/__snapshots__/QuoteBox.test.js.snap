// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteBox rendering component QuoteBox should render 1`] = `
<div
  class="card text-center"
  style={
    Object {
      "width": "30rem",
    }
  }
>
  <div
    class="card-body"
  >
    <blockquote
      class="blockquote"
    >
      <p
        class="mb-0"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer posuere erat a ante.
      </p>
      <br />
      <footer
        class="blockquote-footer"
      >
        Author
      </footer>
    </blockquote>
    <hr />
    <button
      className="btn btn-light"
    >
      <i
        className="fa fa-refresh"
      />
    </button>
  </div>
</div>
`;
